testing/trino-product-tests/src/main/java/io/trino/tests/product/cassandra/TestSelect.java

```diff
@@ -551,7 +551,7 @@ public void testSelectUserDefinedTypeInPrimaryKey()
 
         Consumer<QueryResult> assertion = queryResult -> {
             assertThat(queryResult).hasRowsCount(1);
-            Assertions.assertThat(queryResult.row(0).get(0)).isEqualTo(1);
+            Assertions.assertThat(queryResult.getOnlyValue()).isEqualTo(1);
```

Comment:
Regarding the current CI failure, let's revert changes in this class. The result has more than 1 column and we can't use `getOnlyValue` method.